### ISSUE TITLE
R3BAtima does not load library

### ISSUE THREAD

**Issue by ajedele** [R3BRootGroup/R3BRoot#696]
It seems the R3BAtima was not inputted into the cmake library. Philipp looked into the library needed. But given how old the code is, it is unsure which version to use. Therefore, we would have to import the library make in by hand.

I suggest we either contact Vadim to get the information or at least who used it and if they have a working copy. Or we just remove it.

Right now the code is not useable

**Comment by jose-luis-rs** [comment]
Yes, this is the usual problem when people start doing new implementations and then leave, now we need to find a good candidate to continue this implementation. Volunteers?

**Comment by vadimr3b** [comment]
The problem back than was, that the licence of atima did not allow a publication here, but it might have changed meanwhile.
Therefore all I could do was to store it internal at gsi like stated here:
https://github.com/R3BRootGroup/R3BRoot/pull/290#issuecomment-547446432

**Comment by klenze** [comment]
```/u/land/R3BAtima``` on the lxpool seems to contain the lib which should be linked. 

We should probably discuss with Mr Weick if there is a way to put atima on github or something. 

I would rather not include it in r3broot directly as it is 50k lines of fortran.

**Comment by ajedele** [comment]
I talked to Helmut Weick regarding the atima library. ATIMA is all open source. We can put it on GitHub or wherever else is best.

**Comment by jose-luis-rs** [comment]
Hello, 
Atima (or catima for the c++ version) is already in github
https://github.com/gwm17/catima
